docfx/docfx.json

```diff
@@ -163,6 +163,15 @@
       ],
       "dest": "metadata/Pulumi.F5BigIP"
     },
+    {
+      "src": [
+        {
+          "src": "../../pulumi-github/sdk/dotnet",
+          "files": ["Pulumi.*.csproj"]
+        }
+      ],
+      "dest": "metadata/Pulumi.Github"

```

Comment:
I know this way of spelling GitHub is not the right way, but alas, that's the name we used in our .NET SDK per one of the C# classes that I looked at.